README.md

```diff
@@ -43,6 +56,9 @@ The distributionof the 7 different classifications in the dataset is visualized
 
 There is an issue regarding overfitting of Melanocytic nevi, which will be addressed using image augmentation during the model development process.
 
+### Performance Results
+Overall, the training performance of the model is approximatley 78% while the testing of the model yielded 80%.
```

Comment:
typo on approximately

Reply:
can you also specify that this was accuracy and the number of data points in your test/train set?

Reply:
By the way, you can also create tables in markdown